report/report.go

```diff
@@ -1,21 +1,21 @@
 package report
 
 import (
-	"fmt"
-	"github.com/1Password/dep-report/models"
-	"github.com/1Password/dep-report/versioncontrol"
 	"encoding/json"
-	"github.com/pkg/errors"
-	"log"
 	"os/exec"
 	"strings"
 	"time"
+
+	"github.com/1Password/dep-report/models"
+	"github.com/1Password/dep-report/versioncontrol"
+	"github.com/pkg/errors"
 )
 
 const (
-	GITHUB = "github"
-	GITLAB = "gitlab"
-	GERRIT = "gerrit"
+	GITHUB  = "github"
+	GITLAB  = "gitlab"
+	GERRIT  = "gerrit"
+	UNKNOWN = "unknown/other"
 )
```

Comment:
Should we add "GoModuleCache" to the list while we are here, or is it possible we aren't using the Google-hosted package caches for these?

Reply:
What would "GoModuleCache" mean here? I may not understand what you're going for 🙂 

This enum reflects where the package _source_ is hosted (GitHub, and Google's Gerrit for the vast majority, internal GitLab for some, but exceptions do exist in codebases). So it's not about where it fetches the actual module from, but what the original Git source is.